includes/notes/class-wc-payments-notes-set-up-stripelink.php

```diff
@@ -75,7 +75,7 @@ public static function get_note() {
 		$note       = new $note_class();
 
 		$note->set_title( __( 'Increase conversion at checkout', 'woocommerce-payments' ) );
-		$note->set_content( __( 'Reduce cart abandonment and create a frictionless checkout experience with Link by Stripe. Link autofills your customer’s payment and shipping details so they can check out in just six seconds with the Link optimized experience. That’s 9x faster than shoppers who don’t use Link. Link increases conversion rates by over 7% for logged-in Link customers.', 'woocommerce-payments' ) );
```

Comment:
> That’s 9x faster than shoppers who don’t use Link...

I think the hard numbers will increase the appeal of the overall message and make it more convincing.

Here's the original message rephrased in 250 chars:

`Reduce cart abandonment: create a frictionless checkout experience with Link by Stripe. Link autofills your customer’s payment and shipping details so they can check out 9x faster than non-Link shoppers. Link customers are 7% more likely to convert.`

Reply:
The message's effectiveness can further be increased by providing a link for more details and instructions on how to enable it:

https://woocommerce.com/document/payments/link-by-stripe/

Reply:
@anu-rock the message above was agreed on business side. I don't think we should change it [ [ref](pdjVC1-1aT-p2#comment-1737) ].